webrtc/modules/video_capture/windows/mrc_audio_effect_definition.h

```diff
@@ -0,0 +1,81 @@
+#ifndef WEBRTC_MODULES_VIDEO_CAPTURE_WINDOWS_MRC_AUDIO_EFFECT_DEFINITION_H_
+#define WEBRTC_MODULES_VIDEO_CAPTURE_WINDOWS_MRC_AUDIO_EFFECT_DEFINITION_H_
+
+// This class provides an IAudioEffectDefinition which can be used
+// to configure and create a MixedRealityCaptureAudioEffect
+// object. See https://developer.microsoft.com/en-us/windows/holographic/mixed_reality_capture_for_developers#creating_a_custom_mixed_reality_capture_.28mrc.29_recorder
+// for more information about the effect definition properties.
+
+#define RUNTIMECLASS_MIXEDREALITYCAPTURE_AUDIO_EFFECT L"Windows.Media.MixedRealityCapture.MixedRealityCaptureAudioEffect"
+
+//
+// MixerMode
+// Type: AudioMixerMode as UINT32
+//  0: Mic audio only
+//  1: System audio only
+//  2: Mixing mic and system audio (default)
+//
+#define PROPERTY_MIXERMODE  L"MixerMode"
+
+#include "mrc_effect_template.h"
+#include "webrtc/typedefs.h"
+
+namespace webrtc {
+	namespace videocapturemodule {
+		private enum class AudioMixerMode
+		{
```

Comment:
nit: tabs should be spaces.

Reply:
Fixed for all effect definition classes.